docs/sabakan.md

```diff
@@ -33,19 +53,36 @@ Usage of sabakan:
         Log level [critical,error,warning,info,debug]
   -metrics string
         <Listen IP>:<Port number> (default "0.0.0.0:10081")
+  -sabakan-tls-cert string
+        path to server TLS certificate of sabakan (default "/etc/sabakan/sabakan-tls.crt")
+  -sabakan-tls-key string
+        path to server TLS key of sabakan (default "/etc/sabakan/sabakan-tls.key")
```

Comment:
I think `sabakan-` prefix in the option name and file name is redundant.
It would be better to make it clear that they are for server certificate.